### PR TITLE
Pin GKE cluster version for e2e tests

### PR DESCRIPTION
**Pull request by jmcshane**
# Changes

This change pins the cluster version that will be picked up by kubetest via k8s_extract. Currently, after #818, kubetest will pick up two different --cluster-version flags, and only take the second.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by sbwsg — APPROVED** [review]

**Comment by sbwsg** [comment]
I'm going to go ahead and lgtm this one as well because all the projects are currently blocked by the issue.

/lgtm

**Comment by jmcshane** [comment]
as soon as this is merged ill bump my PR on triggers to validate the change

**Comment by jmcshane** [comment]
@sbwsg 
ah shoot, this is going to need another PR...for some reason hardcoding the version doesn't work with extract:

```
main.go:286: Flag parse failed: invalid argument "1.18.16-gke.502" for "--extract" flag: Unknown extraction strategy: 1.18.16-gke.502
```

**Pull request merged**
